### DISCUSSION TITLE
2.0.0-alpha.2

### DISCUSSION THREAD

**Discussion by JohnNiang** [halo-dev/halo#2545] [General]
> **Warning**
> 当前版本为非正式发布版本，**无法从 `1.5` 和 `1.6` 直接升级到当前版本，也不建议在生产环境使用**，欢迎大家测试并反馈。
> 
> 其次，在 `2.0.0-alpha.1` 中使用的插件在当前版本中可能无法正常运行。
>
> 另外，在 Halo 2.0 正式发布的时候，我们会提供完整的升级指南。

自 [v2.0.0-alpha.1](https://github.com/halo-dev/halo/releases/tag/v2.0.0-alpha.1) 发布以来的变化如下：

## 问题修复

- 修复主题端内容页访问量统计接口角色规则动词配置错误问题 @guqing <https://github.com/halo-dev/halo/pull/2517>
- 修复插件重启引起的异常 @guqing <https://github.com/halo-dev/halo/pull/2518>
- 修复系统默认数据无法正常更新的问题 @JohnNiang <https://github.com/halo-dev/halo/pull/2536>

## 主要特性

- 支持 MySQL 和 Maria DB 作为 Halo 的数据库 @JohnNiang <https://github.com/halo-dev/halo/pull/2512>
- 支持服务端反向代理 Console @JohnNiang <https://github.com/halo-dev/halo/pull/2535>
- 插件卸载支持选择清空数据 @ruibaby
- 菜单项创建支持选择父级菜单项 @ruibaby

## 重要变更

- 重命名后台管理项目 `halo-dev/halo-admin` 为 [halo-dev/console](https://github.com/halo-dev/console) @ruibaby <https://github.com/halo-dev/console/pull/634>
- 重构插件反向代理注册方式 @guqing <https://github.com/halo-dev/halo/pull/2522>
- 优化系统角色模板配置 @guqing <https://github.com/halo-dev/halo/pull/2524>
- 优化系统登录和登出逻辑 @JohnNiang <https://github.com/halo-dev/halo/pull/2528>
- 优化用户登录的逻辑，支持提示错误信息 @ruibaby <https://github.com/halo-dev/console/pull/636>
- 优化 UI 权限控制 @ruibaby

## 依赖变更

- 升级 Spring Boot 至 3.0.0-M5 @JohnNiang <https://github.com/halo-dev/halo/pull/2511>
- 升级 SpringDoc 至 2.0.0-M5 @JohnNiang <https://github.com/halo-dev/halo/pull/2511>

## 其他资源

适用于 2.0 alpha 的插件和主题可查阅：<https://github.com/halo-sigs/awesome-halo>

<hr /><em>This discussion was created from the release <a href='https://github.com/halo-dev/halo/releases/tag/v2.0.0-alpha.2'>2.0.0-alpha.2</a>.</em>

**Comment by ruibaby** [comment]
Wow
